packages/gen-ai-hub/src/client/openai/openai-response.ts

```diff
@@ -0,0 +1,47 @@
+import { HttpResponse } from "@sap-cloud-sdk/http-client";
+import { OpenAiChatCompletionOutput } from "./openai-types.js";
+
+/**
+ * Open AI chat completion response.
+ */
+export class OpenAiChatCompletionResponse {
+  public data: OpenAiChatCompletionOutput
+  constructor(private rawResponse: HttpResponse,) {
+    this.data = rawResponse.data;
+  }
+
+  /**
+   * Usage of tokens in the response.
+   */
+  get usage() {
+    return this.data.usage;
+  }
+  /**
+   * Reason for stopping the model.
+   */
+  get finish_reason() {
```

Comment:
[req] Snake case does not follow our style guide. I would actually suggest to have a function instead of a getter. You could also consider allowing passing an index.